tests/testthat/test-support.R

```diff
@@ -156,6 +156,47 @@ test_that("ll_binom returns a vector of zeros if data missing", {
     rep(0, 5))
 })
 
+test_that("ll_betabinom", {
```

Comment:
Can you add some testing here of the beta binomial distribution:

* perhaps a few key points against some reference implementation e.g., VGAM
* against special case distributions (For α = β = 1, it is the discrete uniform distribution from 0 to n.)
* Similarly, it contains the negative binomial distribution in the limit with large β and n.

Last two from wikipedia!